client/src/main/scala/drt/client/components/TerminalComponent.scala

```diff
@@ -95,11 +96,23 @@ object TerminalComponent {
           <.div(
             <.ul(^.className := "nav nav-tabs",
               <.li(^.className := currentClass, <.a(VdomAttr("data-toggle") := "tab", "Current"), ^.onClick --> {
-                props.router.set(props.terminalPageTab.copy(mode = "current", subMode = subMode, date = None))
+                props.router.set(props.terminalPageTab.copy(
+                  mode = "current",
+                  subMode = subMode,
+                  date = None,
+                  timeRangeStartString = props.terminalPageTab.timeRangeStartString,
```

Comment:
are these two lines copying the existing values?

Reply:
I thought I had a reason for doing this, but I think that reason might have been not paying attention.